### DISCUSSION TITLE
March 1st, 2023

### DISCUSSION THREAD

**Discussion by LukasBuecherl** [SynBioDex/SBOL-Editors#47] [Editors meetings]
# Participants
* Chris Myers
* James Scott-Brown
* Tom Mitchell
* Gonzalo Vidal
* Lukas Buecherl

# Agenda
* JIB submission of specification 3.1.0

## Open Discussion
* Complications with the SBOL specification submission
* JIB office has issues with identifying the right abstract and specification
* @Gonza10V sent correct overleaf project including all the information
* New issue to update the abstract in the specification repository
* Create new issues for other comments on the document (i.e. missing reference)
* Create 3.1.0 tag for repository 

## Website (James S-B) 
*

**Comment by Gonza10V** [comment]
Hi all,
I wanted to proceed and change the specification but I forgot the tasks.
So far I've created a branch to do these changes, [3_1](https://github.com/SynBioDex/SBOL-specification/tree/3_1)
On that branch I've deleted jib_abstract_3_0.tex
Then I've uploaded the new [jib_abstract_3_1.tex](https://github.com/SynBioDex/SBOL-specification/blob/3_1/jib_abstract_3_1.tex) from the Overleaf project.
Here is the PR with the changes: https://github.com/SynBioDex/SBOL-specification/pull/511
I dont fully understand tags so if someone explain how we manage them I'm happy to learn and do them in future.
@tcmitchell is this something that you can explain to me in this discussion? or is this something that you can explain in a chat in next Editors meeting?

**Comment by cjmyers** [comment]
We should create a 3.1.0 release here:

https://github.com/SynBioDex/SBOL-specification/releases

The tag will be created as part of creating the release.
